### ISSUE TITLE
Error creating Jira issue for T1001

### ISSUE THREAD

**Issue by SkiddyBargins** [mvelazc0/attack2jira#15]
I am installing on Jira Software Cloud with it tripping up on the below

[*] Creating Jira issues for ATT&CK's techniques...
        [!] Error creating Jira issue for T1001
{"errorMessages":[],"errors":{"customfield_10038":"Field 'customfield_10038' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10034":"Field 'customfield_10034' cannot be set. 
It is not on the appropriate screen, or unknown.","customfield_10035":"Field 'customfield_10035' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10036":"Field 'customfield_10036' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10037":"Field 'customfield_10037' cannot be set. It is not on the appropriate screen, or unknown."}}

**Comment by mvelazc0** [comment]
Thanks for reporting this @SkiddyBargins . I remember this issue and I thought I fixed it. I will take a look. Just to confirm, are you using the latest release ? Please make sure you are.

**Comment by mvelazc0** [comment]
@SkiddyBargins , I just tested the latest version from master a couple of times and its working for me. As mentioned before, I did see this issue on the previous version. Can you please

1. Make sure you are using the latest version from the master branch
2. If the issue still persists, post the script output here.

Thanks!

![image](https://user-images.githubusercontent.com/4950700/118026023-61772d80-b32e-11eb-8193-702e08f6190d.png)

**Comment by mvelazc0** [comment]
@SkiddyBargins did you test ?

**Comment by SkiddyBargins** [comment]
Yes, 

Getting the below now which looks like the 401 error.
Validated permissions and access.

[!] Success!
[*] Creating the Att&ck project...
[!] Unauthorized. Probably not enough permissions :(

**Comment by SkiddyBargins** [comment]
Looks like a 405 error hitting '/rest/simplified/latest/project'

**Comment by PolarBearGod** [comment]
Just to double check, your project link has the `s` in the protocol of the URL (as in `https`)?

**Comment by PolarBearGod** [comment]
> Looks like a 405 error hitting '/rest/simplified/latest/project'

Just to double check, your project link has the s in the protocol of the URL (as in https)?

**Comment by ScooterSec** [comment]
I'm running into this same issue. I tried running this in a free trial version and everything went smoothly. When trying to run in on my company's instance I got the following error. I have admin rights in both instances. 

`[*] Creating Jira issues for ATT&CK's techniques...
        [!] Error creating Jira issue for T1001
{"errorMessages":[],"errors":{"customfield_10110":"Field 'customfield_10110' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10111":"Field 'customfield_10111' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10112":"Field 'customfield_10112' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10113":"Field 'customfield_10113' cannot be set. It is not on the appropriate screen, or unknown.","customfield_10114":"Field 'customfield_10114' cannot be set. It is not on the appropriate screen, or unknown."}}`

EDIT: I am running the latest version from the master branch
